monkey/infection_monkey/post_breach/actions/communicate_as_new_user.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import random
+import string
+import subprocess
+
+from common.data.post_breach_consts import POST_BREACH_COMMUNICATE_AS_NEW_USER
+from infection_monkey.post_breach.actions.add_user import BackdoorUser
+from infection_monkey.post_breach.pba import PBA
+from infection_monkey.telemetry.post_breach_telem import PostBreachTelem
+from infection_monkey.utils import is_windows_os
+
+CREATED_PROCESS_AS_USER_WINDOWS_FORMAT = "Created process '{}' as user '{}'."
+CREATED_PROCESS_AS_USER_LINUX_FORMAT = "Created process '{}' as user '{}'. Some of the output was '{}'."
+
+USERNAME = "somenewuser"
```

Comment:
I think we can reuse a value from the configuration. `user_to_add`

Reply:
I'd rather not, so they won't clash.